core/src/test/java/io/grpc/ServerProviderTest.java

```diff
@@ -49,6 +49,20 @@ public void unavailableProvider() {
     assertNull(ServerProvider.load(cl));
   }
 
```

Comment:
added extra test

Reply:
changed test to better resemble the OSGi path. The 'default' classloader will fail to work. The OSGi context classloader must take over its job.